libbpf-tools/containers.c

```diff
@@ -17,3 +18,30 @@ struct container get_container_info(int map_fd, uint64_t mtnnsid) {
 
 	return container;
 }
+
+const char *NODE_HEADER = "NODE";
+const char *NAMESPACE_HEADER = "NAMESPACE";
+const char *POD_HEADER = "POD";
+const char *CONTAINER_HEADER = "CONTAINER";
+
+void print_container_info_header(void) {
+	/*
+	 * Format options used are the following:
+	 * '-' is used to left justify the data printed.
+	 * 16 is used to indicate the minimal width of the data printed, if data are
+	 * less than 16, spaces will be added.
+	 * '.16' is used to indicate the maximal number of characters to print, in
+	 * this case 16.
```

Comment:
It's something I'm not sure about. I would rather remove it and print the full length even if the table alignment breaks. Otherwise we'll need to create a way to make sure that the users understand the output was trimmer (like appending (...) or something like that. I think it's a topic for another PR/issue.